anms-ui/public/app/components/adm/Adm.vue

```diff
@@ -12,8 +12,8 @@
           </tr>
         </thead>
         <tbody>
-          <template :key="index" v-for="(adm, index) in adms">
```

Comment:
I don't know if this makes a difference but I think it undoes one of the earlier changes that @d-linko made for other issues.

Reply:
we went over it other upgrades nick needed to make conflicted with those changes so it was needed to change back

Reply:
Yeah, this change back was needed in order to compile the adm.vue component correctly.